src/Http/Controllers/ProductController.php

```diff
@@ -31,16 +32,13 @@ public function show(int $productId)
             'max_sale_qty',
             'qty_increments',
             ...$product->superAttributes->pluck('attribute_code'),
+            ...$product->superAttributes->pluck('attribute_code')->map(fn ($attribute) => "super_{$attribute}"),
+            ...$product->superAttributes->pluck('attribute_code')->map(fn ($attribute) => "super_{$attribute}_values"),
```

Comment:
I'm not sure about this part, it doesn't look very nice and we repeat the same thing in the product searchable trait. Is there any way to make this nicer?